src/Workspaces/Core/Portable/Classification/ClassificationTypeNames.cs

```diff
@@ -28,7 +28,7 @@ public static class ClassificationTypeNames
         public const string TypeParameterName = "type parameter name";
 
         public const string FieldName = "field name";
-        public const string EnumFieldName = "enum field name";
+        public const string EnumMemberName = "enum member name";
```

Comment:
Do we have to update EditorColors.xml in the VS repo for theming?

Reply:
I'm not sure if the right thing happened there when the C# ones were introduced either.

Reply:
We don't need to update EditorColors.xml yet. This is just an API-only change. In VS, these all map back to "identifier" for now. When we introduce this feature for VS we'll update it for theming.